docs/en/observability/apm/manage-storage/ilm-how-to.asciidoc

```diff
@@ -18,18 +18,15 @@ A default policy is applied to each APM data stream, but can be customized depen
 [NOTE]
 ====
 // Explain what changed
-// Before
-Before 8.15, clusters used {ref}/index-lifecycle-management.html[index lifecycle management (ILM)] to provide the default data retention settings for APM data.
-// After
-As of 8.15, new clusters use {ref}/data-stream-lifecycle.html[data stream lifecycle (DSL)] to provide default data retention settings, but the customization for lifecycle is still performed using ILM policies, which are now configured to override the default DSL polices.
+Clusters are managed by {ref}/index-lifecycle-management.html[index lifecycle management (ILM)] to provide the default data retention settings for APM data as well as allow customization for lifecycle.
+
+New clusters created in 8.15 and 8.16 will use {ref}/data-stream-lifecycle.html[data stream lifecycle (DSL)] to provide default data retention settings, but the customization for lifecycle is still performed using ILM policies, which are configured to override the default DSL polices. When these clusters are upgraded to 8.17 the default will revert back to ILM, however, the indices created will continue to use DSL. The changes only affect default lifecycle policies, so, if you have custom ILM policies configured then there are expected to be no changes and the custom ILM policy will continue to be used.
 
 // What this means for existing clusters that are upgraded
-For _existing_ clusters upgrading to 8.15, the default ILM policies should still work as expected.
-You can can continue using ILM or switch the default to DSL explicitly using the {ref}/data-stream-apis.html[data stream API].
+On upgrade to 8.17, _all_ APM indicies will be managed by ILM. This includes the clusters upgrading from 8.15 and 8.16, however, the indicies created during 8.15 and 8.16 will continue to use DSL. If you have custom ILM policies configured, then, there are expected to be NO changes and the custom ILM policies will continue to manage the APM indicies.
```

Comment:
Similar message as above, I would move these details to either release notes or Known issues section. WDYT?

Reply:
Sounds reasonable, let me make the changes.

Reply:
I was about to suggest something similar. 🙂  Maybe something like:

```asciidoc
In the latest version of Elastic APM, clusters are managed by
{ref}/index-lifecycle-management.html[index lifecycle management (ILM)]
to provide the default data retention settings for APM data as well as allow
customization for lifecycle.

[IMPORTANT]
====
*If you created a new cluster in 8.15 or 8.16*, that cluster will use
{ref}/data-stream-lifecycle.html[data stream lifecycle (DSL)] to provide default
data retention settings, but the customization for lifecycle is still performed
using ILM policies, which are configured to override the default DSL polices.

Read more in <<apm-known-issues>>.
====
```

Reply:
@simitt I have updated with the above suggestion, let me know which one sounds better.

Reply:
Reworded a bit more after slack discussion. Basically ALL clusters will move to ILM after upgrade to 8.17, however, some indices which were created in 8.{15, 16} will be with DSL (not the new indices created post upgrade).